camel_test.go

```diff
@@ -47,6 +48,7 @@ func TestLowerCamelCase(t *testing.T) {
 		"skip   many spaces":    "skipManySpaces",
 		"skip---many-dashes":    "skipManyDashes",
 		"skip___many_underline": "skipManyUnderline",
+		"XRequestId":            "xRequestId",
```

Comment:
Comment as for upper case:
```suggestion
		"XRequestId":            "xRequestId",
		"HTTPStatusCode":        "httpStatusCode",
```